### PR TITLE
[RFR] react app + deployment

### PR DESCRIPTION
**Pull request by pemoreau**
- [x] bootstrap react
- [x] make tests pass
- [x] run client in a docker container

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.App {
```
Comment (jdemangeon) [inline src/App.css]: This file is not used anymore, i'm right? So delete it

File: src/index.js [hunk 2]
```diff
@@ -1 +1,12 @@
-console.log('Hello wolrd');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```
Comment (jdemangeon) [inline src/index.js]: This is working now?

File: src/logo.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (jdemangeon) [inline src/logo.svg]: You can remove this file

File: src/index.js (hunk 2 above)
Comment (pemoreau) [reply src/index.js]: Yes this works since index.js is not tested by Jest

**Pull request merged**